Build failure

Got this on a dependabot update PR from 5.0.0 to 5.0.1

```
Error: /home/runner/.nuget/packages/fastexpressioncompiler.internal.src/5.0.1/contentFiles/cs/netstandard2.0/FastExpressionCompiler/FastExpressionCompiler.cs(1009,29): error CS0649: Field 'ExpressionCompiler.NestedLambdaForNonPassedParams.NonPassedParams' is never assigned to, and will always have its default value null 
```

--- a/src/FastExpressionCompiler/FastExpressionCompiler.cs
+++ b/src/FastExpressionCompiler/FastExpressionCompiler.cs
@@ -1005,8 +1005,10 @@ namespace FastExpressionCompiler
             public static FieldInfo NonPassedParamsField =
                 typeof(NestedLambdaForNonPassedParams).GetField(nameof(NonPassedParams));
 
+#pragma warning disable CS0649
             public readonly object NestedLambda;
             public object[] NonPassedParams;
+#pragma warning restore CS0649
             public NestedLambdaForNonPassedParams(object nestedLambda) => NestedLambda = nestedLambda;
         }
 

